src/nigui/private/windows/windows.nim

```diff
@@ -127,10 +134,13 @@ const
   WS_EX_CLIENTEDGE* = 0x00000200
   WS_GROUP* = 0x00020000
   WS_HSCROLL* = 0x00100000
+  WS_MAXIMIZEBOX* = 0x00010000
+  WS_MINIMIZEBOX* = 0x00020000
```

Comment:
Some of these new constants aren't really relevant for the checkbox but I figured they'll end up getting used for things like non-resizable windows. That's what I used these for but wasn't sure how to implement them as an option for a window - I used it for all new windows in my local version.